ephios/core/templates/core/group_list.html

```diff
@@ -14,25 +14,54 @@ <h1>{% trans "Groups" %}</h1>
         <a class="btn btn-secondary mb-3" href="{% url "core:group_add" %}"><span
                 class="fa fa-plus"></span> {% trans "Add group" %}</a>
     {% endif %}
-    <table id="event_table" class="table table-striped display">
-        <thead>
-        <tr>
-            <th>{% trans "Name" %}</th>
-            <th>{% trans "Action" %}</th>
-        </tr>
-        </thead>
-        <tbody>
+    <ul class="list-group mb-3">
         {% for group in group_list %}
-            <tr>
-                <td>{{ group.name }}</td>
-                <td class="d-flex">
-                    <a class="btn btn-secondary btn-sm text-nowrap" href="{% url "core:group_edit" group.pk %}"><span
-                            class="fa fa-edit"></span> <span class="d-none d-md-inline">{% trans "Edit" %}</span></a>
-                    <a class="btn btn-secondary btn-sm text-nowrap ms-1" href="{% url "core:group_delete" group.pk %}"><span
-                            class="fa fa-trash-alt"></span> <span class="d-none d-md-inline">{% trans "Delete" %}</span></a>
-                </td>
-            </tr>
+            <li class="list-group-item">
+                <div class="row mb-0">
+                    <div class="col-12 col-lg-4">
+                        <h5 class="d-inline">
+                            {{ group.name }}
+                        </h5>
+                        <div id="popover-title-{{ group.id }}" class="d-none"></div>
+                        <div id="popover-content-{{ group.id }}" class="d-none">
+                            {% for user in group.user_set.all %}{% if not forloop.first %}, {% endif %}
+                                {{ user.get_full_name }}{% endfor %}
+                        </div>
+                        <span class="float-end pe-1"
+                              tabindex="0"
+                              data-bs-toggle="popover"
+                              data-bs-trigger="hover"
+                              data-bs-placement="right"
+                              data-bs-title-ref="#popover-title-{{ group.id }}"
+                              data-bs-content-ref="#popover-content-{{ group.id }}"
+                        >
+                            <span class="fas fa-users"></span>
+                            {{ group.user_count }}
+                        </span>
+                    </div>
+                    <div class="col">
+                        {% if group.is_planning_group %}
+                            <span class="badge text-bg-light">{% translate "add events" %}</span>
+                        {% endif %}
+                        {% if group.is_hr_group %}
+                            <span class="badge text-bg-secondary">{% translate "edit users" %}</span>
+                        {% endif %}
+                        {% if group.is_management_group %}
+                            <span class="badge text-bg-dark">{% translate "change permissions" %}</span>
```

Comment:
I don't like that we are introducing another term for the same concept, I think thats confusing

Reply:
Again, I take suggestions. The label from the group form is too long (change qualifications and permissions).
I also tried putting multiple labels but it felt weird having multiple labels per checkbox in the form.

Reply:
hm fair. We are not using the term managemant group towards the user yet?

Reply:
I'm more happy with the new help text now